primitives/bignumber.go

```diff
@@ -552,10 +585,13 @@ func (u *U256) LTE(v *U256) (r int) {
 	return
 }
 
-func (u U256) toJSON() string {
+func (u U256) ToJSON() string {
 	u1 := big.Int(u)
 	innerArrOfU := u1.Bytes()
 	var test = make([]byte, 2*len(innerArrOfU))
 	hex.Encode(test, innerArrOfU[:])
+	if string(test) == "" {
```

Comment:
When the `test` became `""`?
Since the content of the `test` is hex-encoding of integer, shouldn't it have some value always?